src/ratgdo.cpp

```diff
@@ -53,6 +55,9 @@ void setup() {
 
     setup_comms();
 
+    // We can set current lock state to unknown as HomeKit has value for that.
+    // But we can't do the same for door state as HomeKit has no value for that.
+    garage_door.current_lock = CURR_UNKNOWN;
```

Comment:
I think we just need to send a status command on boot to get the proper states

Reply:
We can back this out if you figure out how to get the status during initialization (fyi, I moved it from ratgdo.cpp into homekit.cpp).